app-backend/app/src/main/resources/application.conf

```diff
@@ -41,6 +41,12 @@ auth0 {
   # Auth0 Bearer with proper permissions for Management API
   bearer = ""
   bearer = ${?AUTH0_MANAGEMENT_BEARER}
+
+  managementClientId = ""
+  managementClientId = ${?AUTH0_MANAGEMENT_CLIENT_ID}
```

Comment:
I think that I captured the necessary details in https://github.com/azavea/raster-foundry/issues/1117#issuecomment-282097360.

I went through and created Auth0 management API clients for each domain. They're all named `Web Management API`.